### ISSUE TITLE
Compiling stopped working in 1.6.4 in VSCode

### ISSUE THREAD

**Issue by ZacBoyet** [johan-v-r/LibSassBuilder#38]
I develop in vscode using "dotnet watch run" and in the 1.6.4 version all compilation has stopped working. If I revert back to 1.6.3 it all starts working again.

Here is the csproj/project
https://github.com/StableCube/Bulzor/blob/master/Demo/Server/Demo.Server.csproj

Any ideas why this may be happening?

Thanks

**Comment by johan-v-r** [comment]
Hi @ZacBoyet  - can you test again with v2.0.1
Hoping it's related to #40

**Comment by ZacBoyet** [comment]
I have since discovered more of what my problem was. Previously I was using ProjectReference to reference classlib projects with sass in them that I wanted compiled. That seemed to work fine out of the box in 1.6.3 but stopped working after that. The solution I landed on was to add LibSassBuilder to all my classlibs and that seemed to get the compilation going again.

It does appear that the 2.0.1 update gets things in ProjectReference working again!

Thanks for the update!
